src/app/app.module.ts

```diff
@@ -1,16 +1,24 @@
 import { NgModule } from '@angular/core';
```

Comment:
we could be using standalone components instead of these. modules nowadays are added complexities except possibly:

- if you have child components.. I'm not sure if there is a usecase for mixed standalone and non-standalone
- if you have services that are *not* provided in root.